README.md

```diff
@@ -35,6 +35,8 @@ Choose a language:
 * [C#](./docs/getting-started/csharp.md)
 * [Go](./docs/getting-started/go.md)
 
+> **Hands-on:** Try the tutorials in the [CDK for Terraform](https://learn.hashicorp.com/collections/terraform/cdktf) collection on HashiCorp Learn.
+
```

Comment:
is the Markdown quote intentionally?

Reply:
yep, that's usually what we do in docs to draw attention to it, but we can remove it if you feel strongly against it 😄 

![image](https://user-images.githubusercontent.com/7556325/121140431-b3f12000-c7ee-11eb-9444-dd2e05af6877.png)